core/input.ts

```diff
@@ -290,9 +290,6 @@ export class Input {
     if (this.connection) {
       this.connection.dispose();
     }
-    // AnyDuringMigration because:  Type 'null' is not assignable to type
```

Comment:
I can't leave a comment up there but on line 66 `getSourceBlock` type can be changed so that it can't return null anymore.